tests/test_cases/issue_957/issue_957.py

```diff
@@ -5,7 +5,7 @@
 def wait_edge(dut):
     # this trigger never fires
     yield First(
-        RisingEdge(dut.and_output)
+        RisingEdge(dut.stream_out_ready)
```

Comment:
While this fixes the error "``and_output`` not found in design" maybe we should align the VHDL with the Verilog version of the design. There is at least one other such problem in the tests (for a different signal).

Reply:
Done, #1396.